src/internal/connector/onedrive/drive_test.go

```diff
@@ -277,6 +277,46 @@ func (suite *OneDriveUnitSuite) TestDrives() {
 	}
 }
 
+func (suite *OneDriveUnitSuite) Test_driveName() {
+	_, flush := tester.NewContext()
+	defer flush()
+
+	webURL := "https://<site name>/personal/<user>_onmicrosoft_com/Documents"
```

Comment:
have you happened to come across any documentation that shows the drive name is always the last path element? I know we've seen that some other parts of the URL can be changed by the admin so I thought I'd ask